docs/setup-and-installing.md

```diff
@@ -102,36 +124,48 @@ Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [
 
 <%(/Expandable)%>
 
-<%(Callout type="alert" message="
-* You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment. In the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux.<br/>
-* You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
-* You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
-* Unity gives you build support for your development machine (Windows or Mac) by default.")%>
+#### Make sure you have chosen the right build support
 
-<%(#Expandable title="Having problems?")%>
-If you need help using the GDK, please come and talk to us about the software and the documentation via:
+- You need **Linux** build support. This is because server-workers in a cloud deployment run in a Linux environment.<br/>
+- You need **Mac** build support if you are developing on a Windows PC and want to share your game with Mac users.<br/>
+- You need **Windows** build support if you are developing on a Mac and want to share your game with Windows PC users. <br/>
+- Unity gives you build support for your development machine (Windows or Mac) by default.
 
-**The SpatialOS forums**
+#### Need some help?
 
-Visit the [support section](https://forums.improbable.io/new-topic?category=Support&tags=unity-gdk) in our forums and use the unity-gdk tag.
+If you need help using the GDK, come and talk to us about the software and the documentation via:
 
-**Discord**
+- **The SpatialOS forums** - Visit the [support section](https://forums.improbable.io/new-topic?category=Support&tags=unity-gdk) in our forums and use the unity-gdk tag.
+- **Discord** - Find us in the [#unity channel](https://discord.gg/SCZTCYm). You may need to grab Discord [here](https://discordapp.com/).
+- **Github issues** - Create an [issue](https://github.com/spatialos/gdk-for-unity/issues) in this repository.
+  <br/>
+  <br/>
 
-Find us in the [#unity channel](https://discord.gg/SCZTCYm). You may need to grab Discord [here](https://discordapp.com/).
+### Clone the SpatialOS GDK for Unity repository
 
-**Github issues**
+To run the GDK for Unity, you need to download the source code. To do this, you need to clone the GDK repository.
 
-Create an [issue](https://github.com/spatialos/gdk-for-unity/issues) in this repository.
-<%(/Expandable)%>
+#### Using a terminal:
 
-### Clone the SpatialOS GDK for Unity repository
+Clone the SpacialOS GDK for Unity repository using one of the following terminal commands:
```

Comment:
```suggestion
Clone the SpatialOS GDK for Unity repository using one of the following terminal commands:
```

could we also reverse the ordering of SSH and HTTPS? the recommended one is HTTPS